odoo_backup_sh/static/description/index.html

```diff
@@ -0,0 +1,191 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Odoo Backups </h2>
```

Comment:
It must be the same as Name in the Manifest. 
I think to use "odoo-backup.sh" as a brand name for the service. Let me know if you have better idea

Reply:
It was ignored